code/modules/client/client_procs.dm

```diff
@@ -59,7 +59,7 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 			var/msg = "Your previous action was ignored because you've done too many in a minute."
 			if (minute != topiclimiter[ADMINSWARNED_AT]) //only one admin message per-minute. (if they spam the admins can just boot/ban them)
 				topiclimiter[ADMINSWARNED_AT] = minute
-				msg += " Administrators have been informed."
+				msg += "If you think these alerts are a mistake, possible fixes could be: 1. Disconnecting any controllers 2. Upgrading byond version 3. Clearing cache."
 				log_game("[key_name(src)] Has hit the per-minute topic limit of [mtl] topic calls in a given game minute")
 				message_admins("[ADMIN_LOOKUPFLW(usr)] [ADMIN_KICK(usr)] Has hit the per-minute topic limit of [mtl] topic calls in a given game minute")
 			to_chat(src, span_danger("[msg]"))
```

Comment:
sorry to do this to you but this type of string concatentation wasn't great beforehand and it's even uglier now. let's do it like this

```dm
var/message_list = list("Your previous action was ignored because you've done too many in a minute.")
if (minute != topiclimiter[ADMINSWARNED_AT]) //only one admin message per-minute. (if they spam the admins can just boot/ban them)
	topiclimiter[ADMINSWARNED_AT] = minute
	message_list += "If you think these alerts are a mistake, possible fixes could be:"
	message_list += "1. Disconnecting any controllers."
	message_list += "2. Upgrading your BYOND version."
	message_list += "3. Clearing your BYOND cache."
	log_game("[key_name(src)] has hit the per-minute topic limit of [mtl] topic calls in a given game minute")
	message_admins("[ADMIN_LOOKUPFLW(usr)] [ADMIN_KICK(usr)] has hit the per-minute topic limit of [mtl] topic calls in a given game minute")

to_chat(src, span_danger(message_list.Join("\n"))
```

i cleaned up some of the comments to look better in the numbered-point list (and the log/admins message) but we wanna use `Join()` on lists when we do string concatenation list this because otherwise the string tree starts to go womp womp - it doesn't matter _as_ much here but it's just a better practice and it allows us to easily put in newlines without having to inject `\n` everywhere in those messages or have that ugly ass leading-space (which you accidentally removed btw)